packages/blocks/databricks/src/lib/actions/execute-sql-statement.ts

```diff
@@ -0,0 +1,109 @@
+import { createAction, Property, Validators } from '@openops/blocks-framework';
+import { makeHttpRequest } from '@openops/common';
+import { AxiosHeaders } from 'axios';
+import { databricksAuth } from '../common/auth';
+import { getDatabricksToken } from '../common/get-databricks-token';
+import { DatabricksSqlExecutionResult } from '../common/sql-execution-result';
+import { warehouseId } from '../common/warehouse-id';
+import { workspaceDeploymentName } from '../common/workspace-deployment-name';
+
+const MAX_QUERY_TIMEOUT_SECONDS = 50;
+const RETRY_TIMEOUT = 15000;
```

Comment:
Maybe add "_MILLISECONDS" to the end of it, I find it a bit confusing that the top one specifies the time unit and this one doesnt